packages/nimble-components/src/wafer-map/specs/features/rendering.md

```diff
@@ -0,0 +1,189 @@
+# Rendering++
+
+## Problem Statement
+
+The problem we are addressing is the need for faster rendering in our application. Currently, the rendering process is slow and affects the overall performance of the system. This issue is particularly noticeable when dealing with large or complex datasets. Therefore, we need to design a solution that optimizes the rendering process and improves the user experience.
+
+The proposed design should consider the following factors:
+
+-   Efficiently handle large and complex datasets
```

Comment:
What makes a dataset "complex"? Lots of metadata? Non-uniform values? Unsorted values?